### ISSUE TITLE
🧪 [E2E test] File

### ISSUE THREAD

**Issue by pusztaienike** [SenseNet/sn-client#951]
# 🧪E2E test cases

The scope of these tests is to ensure that file creation, modification, and delete works as it is intended.

## 😎 Role

All test should run as admin.

# Test case 1

## 🧫 Purpose of the test

To ensure that file creation work properly.

## 🐾 Steps

1. Login with admin role
2. Click on Content menuitem
3. Click on IT Workspace in the tree
4. Click on Document Library (under IT Workspace) in the tree
5. Click on 'Add new' button
6. Select File from the dropdown list
7. Fill the form with the following data:
- Filename: TestFile
8. **Step:** Click on submit button
    **Expected result:** TestFile should be in the list

![image](https://user-images.githubusercontent.com/22347393/94244671-83e04600-ff19-11ea-923c-1cc7fa30ffcd.png)

# Test case 2

## 🧫 Purpose of the test

To ensure that modifying a file works properly

## 🐾 Steps

1. Right click on TestFile
2. Select Edit from the dropdown list
3. Change the FileName from TestFile to ChangedTestFile
4. **Step:** Click on submit button
    **Expected result:** ChangedTestFile should be in the list, TestFile should not be

![image](https://user-images.githubusercontent.com/22347393/94244851-c6a21e00-ff19-11ea-9bcd-caddd2e81bb0.png)

# Test case 3

## 🧫 Purpose of the test

To ensure that delete a file content works properly

## 🐾 Steps

1. Right click on ChangedTestFile
2. Select Delete from the dropdown list
3. Tick permanently
4. **Step:** Click on Delete button
    **Expected result:** ChangedTestFile should not be in the list

**Comment by fillerInk** [comment]
I'd like to work on this issue

**Comment by fillerInk** [comment]
Excuse me if this is trivial, but how do I log in with the admin role? The two demo accounts I'm getting after running locally are `businesscat` and `devdog`

**Comment by herflis** [comment]
Hi @fillerInk, `businesscat` is the admin in this case . Thank you for reporting the lack of information, I'll double-check the guides. :)

**Comment by fillerInk** [comment]
Thank you @herflis!

I'm getting an error while trying to use the `cy.login()` command defined at [apps/sensenet/cypress/support/commands.js](https://github.com/SenseNet/sn-client/blob/d52fee4a577cc2a3f428b8d33109b2b14a9ae7c5/apps/sensenet/cypress/support/commands.js#L42)

The error message I recieved 

```
cy.request() failed on:

https://is.demo.sensenet.com/connect/token

The response we received from your web server was:

  > 400: Bad Request

This was considered a failure because the status code was not 2xx or 3xx.

If you do not want status codes to cause failures pass the option: failOnStatusCode: false
```

I'm getting the same error while running the other tests that are already in the repo. Where have I gone wrong?

**Comment by herflis** [comment]
@fillerInk can you please share here your config (cypress.json)?

**Comment by herflis** [comment]
@fillerInk See the attached image to make sure everything is set up correctly in the config
![image](https://user-images.githubusercontent.com/6968860/95084976-e823b680-071e-11eb-8064-f6b6252cddd4.png)

**Comment by fillerInk** [comment]
This solved the issue! Do you think it'll be appropriate to add this image to the instructions in the [CONTRIBUTING.md](https://github.com/SenseNet/sn-client/blob/develop/apps/sensenet/cypress/CONTRIBUTING.md) file inside the apps/sensenet/cypress directory?

**Comment by herflis** [comment]
That was the plan, I was just waiting to be confirmed that it really helps :) I'm glad that your problem is solved. Thanks for the feedback

**Comment by herflis** [comment]
Hi @fillerInk ,  did you manage to deal with this? Is there anything we can help you with? :)

**Comment by fillerInk** [comment]
I have got started with Cypress but couldn't figure out how to access a DOM element that appears only after the page is loaded, but the test goes on before the page has loaded completely

**Comment by herflis** [comment]
Hi @fillerInk take a look at the tests that have been already done and see if it helps. [https://github.com/SenseNet/sn-client/tree/develop/apps/sensenet/cypress/integration](https://github.com/SenseNet/sn-client/tree/develop/apps/sensenet/cypress/integration)
